From: Daniel Wagner <wagi@monom.org>
Date: Thu, 14 Dec 2023 09:36:21 +0100
Subject: [PATCH 354/354] Linux 4.19.299-rt130 REBASE
Origin: https://git.kernel.org/cgit/linux/kernel/git/rt/linux-stable-rt.git/commit?id=80655f93cbb6c67274503b1b10a3826384a97765

Signed-off-by: Daniel Wagner <wagi@monom.org>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 1199ebade17b..6fa797e5b850 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt16
+-rt130
